### PR TITLE
[ui-admin][improvement] Rename AlertComponent to GlobalAlert, MetricsComponent to Metrics and create a separate directory for the admin pages

### PR DESCRIPTION
**Pull request by ananya-agarwal**
## What changes were proposed in this pull request?

This PR is about:

1. Renaming admin page's MetricsComponent to Metrics to maintain consistency with all other components and removing redundancy.

2. Shifting the admin pages (already created metricsComponent now Metrics) to the new location of admin page (to be created in the PR) i.e. from js/apps/components/reactComponents to js/apps/admin and js/apps/admin/Metrics

3. Renaming AlertComponent to GlobalAlert for making consistency with other components (without changing its location and keeping it as js/apps/components/reactComponents)

## How was this patch tested?
Manual testing and checking UI looks fine. This is just an improvement PR.

Please review [Hue Contributing Guide](https://github.com/cloudera/hue/blob/master/CONTRIBUTING.md) before opening a pull request.

### CODE REVIEW

File: desktop/core/src/desktop/templates/metrics.mako
Comment (ananya-agarwal) [inline desktop/core/src/desktop/templates/metrics.mako:1]: There was some extra code from the metricsComponent that was mistakenly left during removing the metrics.mako code (a script) while changing the metrics code from mako to reactJS. I am doing that cleanup also within a commit in this PR.

**Review by JohanAhlen — APPROVED** [review]
LGTM!

**Review by tabraiz12 — APPROVED** [review]
Renaming Looks good, verify all the basic functionality around editor, jobBrowser once.

**Review by bjornalm — APPROVED** [review]
Nice work. I agree with @tabraiz12, we need to manually verify that everything still works. Even simple refactoring tasks like renaming can potentially break functionality if we miss something.

**Pull request merged**